Fix bottom right area of sprites being cut off

Piepow opened the pull request:
This reverts commit 8eaa040e3ed2a7a40fdb21eec284d85c2540505e. For some reason the commit causing the issue isn't in 0.6, but it is in 0.7.

I originally looked into it because the skins looked smushed to the right, which made the tees look a lot worse than in 0.6.

### Before
![before](https://user-images.githubusercontent.com/20409621/84224295-20392680-aaaa-11ea-8c52-f348b4c229b4.png)

### After
![after](https://user-images.githubusercontent.com/20409621/84224315-31823300-aaaa-11ea-85d2-a7262553de51.png)

---

### Before
![before_m](https://user-images.githubusercontent.com/20409621/84224340-4363d600-aaaa-11ea-8ce3-24b430d10908.png)

### After
![after_m](https://user-images.githubusercontent.com/20409621/84224345-46f75d00-aaaa-11ea-8e18-43d5fa1d66ab.png)

Piepow committed:
Revert "fixed sprite rendering. Closes #44"

This reverts commit 8eaa040e3ed2a7a40fdb21eec284d85c2540505e.

fokkonaut commented:
> For some reason the commit causing the issue isn't in 0.6, but it is in 0.7.
Thats because its from 2015. 0.7 is in development since way before that

Sonix- commented:
In the after screen the hud cursor, a line appears. Gotta check if the game.png is faulty and , if this gets accepted, probably fix it.

Piepow committed:
Shrink sprite texture coordinates

- Fixes bug where pixels from outside the sprite border are rendered
- Code by Jupeyy

Piepow commented:
> In the after screen the hud cursor, a line appears. Gotta check if the game.png is faulty and , if this gets accepted, probably fix it.

Nice catch, I checked the game.png but it seemed to be fine.
![2020-06-10T20:44:36-04:00](https://user-images.githubusercontent.com/20409621/84346451-010bc900-ab7e-11ea-9c47-ba50337a8f80.png)

I looked into it more and found that DDNet used to have the same problem: https://github.com/ddnet/ddnet/issues/834. This seemed to be fixed by https://github.com/ddnet/ddnet/commit/4fc7187a1496ae7ca51faa11f8bf46e536e1a10c, so I used that code and it worked fine.

### Before
![screenshot_2020-06-11_00-39-42](https://user-images.githubusercontent.com/20409621/84346681-88f1d300-ab7e-11ea-8a14-00ebd9fc6011.png)

### After
![screenshot_2020-06-11_00-39-44](https://user-images.githubusercontent.com/20409621/84346690-8c855a00-ab7e-11ea-8568-6fca0082d00f.png)
